docs/_includes/ds6/dialog.html

```diff
@@ -396,6 +396,47 @@ <h2 class="large-text-1 bold-text" id="dialog-title">Heading</h2>
             <p><a href="http://www.ebay.com">www.ebay.com</a></p>
         </div>
     </div>
+</div>
+        {% endhighlight %}
+    </div>
+
+    <h3>Maskless Dialog</h3>
```

Comment:
I think for now we should just add this modifier to the fullscreen example and add a one-line note there about it's use. Not sure at this point if we should promote it's use elsewhere.